code/modules/research/designs/bluespace_designs.dm

```diff
@@ -90,3 +90,13 @@
 	materials = list(MAT_METAL = 75000, MAT_GLASS = 37500, MAT_SILVER = 3000)
 	build_path = /obj/item/rpd/bluespace
 	category = list("Bluespace")
+
+/datum/design/bluespaceshotglass
+	name = "Bluespace Shot Glass"
+	desc = "For when you need to make the Bartender's life extra hell."
+	req_tech = list("bluespace" = 3, "toxins" = 6)
```

Comment:
Not an error just a suggestion, I'd make the `req_tech` equal to what "bluespace beaker" has. The "toxins" level during a round is either level 1 or level 7 (scientists either do it or don't).

Feel free to dismiss this if you want to keep it as is, it's just a heads up.